packages/components/src/Lettera/Models.purs

```diff
@@ -18,22 +33,101 @@ isPreviewArticle :: FullArticle -> Boolean
 isPreviewArticle (PreviewArticle _) = true
 isPreviewArticle _ = false
 
-type ArticleStub =
-  { title    :: String
-  , uuid     :: String
-  , preamble :: String
+newtype LocalDateTime = LocalDateTime DateTime
+
+fromUTCTime :: DateTime -> Effect (Maybe LocalDateTime)
```

Comment:
Do we need a comment here? 🤔

Reply:
Yes I think we need to mention about the timezone adaptation explicitly, even though the name is a hint.